### PR TITLE
[GVBC] Enabling multiple legend selection for Grouped Vertical Bar Chart

### PR DESCRIPTION
**Pull request by srmukher**
Enabling multiple legend selection for Grouped Vertical Bar Chart

image

### CODE REVIEW

File: change/@fluentui-react-charting-d99f14e6-dd3a-4f33-a8b3-9e28ea5bcfab.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-d99f14e6-dd3a-4f33-a8b3-9e28ea5bcfab.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33511&commitId=c24d9e57e18101c11b6e998b4b40036e3768b88f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/charts/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx [hunk 2]
```diff
@@ -109,7 +110,7 @@ export class GroupedVerticalBarChartBase extends React.Component<
       dataForHoverCard: 0,
       isCalloutVisible: false,
       refSelected: null,
-      selectedLegend: props.legendProps?.selectedLegend ?? '',
+      selectedLegends: [],
```
Comment (AtishayMsft) [inline packages/charts/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx:113]: >[] [](http://example.com/codeflow?start=23&length=2)

same as other PRs

Comment (srmukher) [reply packages/charts/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx:113]: done

Comment (srmukher) [reply packages/charts/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**